src/components/central-program/central-program.scss

```diff
@@ -1,26 +1,31 @@
-$overview-table-bg-color: rgba(245,247,254,1);
+$overview-table-bg-color: rgba(245, 247, 254, 1);
 
 .central-program-page-template {
+  .program-data-overview {
+    .heading {
+      thead th {
+        border-top: 0px;
+        padding-left: 0px;
+        padding-right: 0px;
+      }
+    }
 
-    .program-data-overview {
-        .heading {
-            thead th {
-                border-top: 0px;
-                padding-left: 0px;
-                padding-right: 0px;
-            }
-        }
-
-        table.overview {
-            background-color: $overview-table-bg-color;
+    table.overview {
+      background-color: $overview-table-bg-color;
 
-            td, th {
-                border: 0px;
-            }
+      td,
+      th {
+        border: 0px;
+      }
 
-            div.value {
-                font-weight: normal;
-            }
-        }
+      div.value {
+        font-weight: normal;
+      }
     }
+  }
+}
+
+.scroll-widget {
+  position: fixed;
+  right: 5%;
```

Comment:
Is it possible to get the cursor to turn into a pointer on the dot things? I guess this means it's not an accessible link. :/ But maybe there's not much we can do about that given the design of this widget.

Reply:
Yep! Just added another commit to make the cursor a pointer 👍